spacewalk/setup/spacewalk-setup.spec

```diff
@@ -206,26 +188,22 @@ install -Dd -m 0755 %{buildroot}%{_prefix}/share/salt-formulas/states
 install -Dd -m 0755 %{buildroot}%{_prefix}/share/salt-formulas/metadata
 
 %post
+if [ -e /etc/tomcat/server.xml ]; then
+    cp /etc/tomcat/server.xml /etc/tomcat/server.xml.post-script-backup
+    xsltproc %{_datadir}/spacewalk/setup/server.xml.xsl /etc/tomcat/server.xml.post-script-backup > /etc/tomcat/server.xml
+fi
```

Comment:
Is this ok, if this happens every time again on every package update ?

Reply:
At least it would overwrite again and again the backup file. I think this is not wanted.

Reply:
right, thans for noticing

Reply:
I created 2 xsl files, one is executed just during a new installation, another only during upgrade